prombench/manifests/prombench/benchmark/3b_prometheus-test_deployment.yaml

```diff
@@ -64,14 +64,24 @@ spec:
           failureThreshold: 30
         command: ["/usr/bin/prometheus"]
         args: [
+          # default prombench args
           "--web.external-url=http://{{ .DOMAIN_NAME }}/{{ .PR_NUMBER }}/prometheus-pr",
           "--storage.tsdb.path=/prometheus",
           "--web.console.libraries=/usr/bin/console_libraries",
           "--web.console.templates=/usr/bin/consoles",
           "--config.file=/etc/prometheus/prometheus.yml",
           # JSON log format is needed for GKE to display log levels correctly.
           "--log.format=json",
-          "--log.level=debug"
+          "--log.level=debug",
```

Comment:
Would be lovely to maybe parametrized the upstream - but we can do that later, LGTM for now.

Reply:
Do you mind create issue on this repo about it?

Reply:
@TheSpiritXIII can you add your thoughts about using Kustomize to have a sync between prometheus-engine collector config and the prombench collector configs here.